app-backend/db/src/main/scala/com/azavea/rf/database/AccessControlRuleDao.scala

```diff
@@ -180,7 +180,7 @@ object AccessControlRuleDao extends Dao[AccessControlRule] {
       )
     """
     .query[String]
-    .list
+    .to[List]
```

Comment:
what's the difference between `.list` and `.to[List]`?

Reply:
One has a deprecation warning attached to it

Reply:
Practically I have no idea